### DISCUSSION TITLE
How i can evit move top on dialog close?

### DISCUSSION THREAD

**Discussion by Julian1993ARG** [serenity-is/Serenity#7041] [Help]
I am utilizing fullCalendar to display a 12-month view. When a user opens a dialog in the middle of the calendar and subsequently closes it, the page automatically scrolls to the top. How can I prevent this unwanted behavior?

**Comment by patriciozeus** [comment]
Hola Julian

No se si te servira esto,  la instruccion que esta remarcada te permite abrir el dialogo en plana mayor.


namespace LDES.ServiciosOms {

    @Serenity.Decorators.registerClass()
    **_@Serenity.Decorators.panel()_**
    export class PgmCotizacionEnProcesoDialog extends Serenity.EntityDialog<PgmCotizacionEnProcesoRow, any> {
        protected getFormKey() { return PgmCotizacionEnProcesoForm.formKey; }
        protected getIdProperty() { return PgmCotizacionEnProcesoRow.idProperty; }
        protected getLocalTextPrefix() { return PgmCotizacionEnProcesoRow.localTextPrefix; }
        protected getNameProperty() { return PgmCotizacionEnProcesoRow.nameProperty; }
        protected getService() { return PgmCotizacionEnProcesoService.baseUrl; }
        protected getDeletePermission() { return PgmCotizacionEnProcesoRow.deletePermission; }
        protected getInsertPermission() { return PgmCotizacionEnProcesoRow.insertPermission; }
        protected getUpdatePermission() { return PgmCotizacionEnProcesoRow.updatePermission; }

        protected form = new PgmCotizacionEnProcesoForm(this.idPrefix);


        protected updateTitle(): void {
            super.updateTitle();
            this.dialogTitle = 'Cotizacion en Proceso';
        }


atte.

Patricio
